sdk/keyvault/perf-tests/keyvault-keys/test/cryptography/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./decrypt.spec";
```

Comment:
I do not expect those perf tests to be used elsewhere, so is there a reason for creating an `index.ts`?

Reply:
Personally I like having barrel files whenever I need to import things from a directory - but you have me thinking and now I'm curious: what's the argument against having them?

Reply:
It is my understanding that the main reason for an index file to exist is if the package will be imported. I just can not think of a scenario where this will be the case.

Reply:
In this case I added it to make this import cleaner and single line: https://github.com/maorleger/azure-sdk-for-js/blob/kv-perf-tests/sdk/keyvault/perf-tests/keyvault-keys/test/index.spec.ts#L3 

But I can remove it as well, don't feel too strongly about it

Reply:
Me neither...